VocalRaidAssistant.lua

```diff
@@ -165,10 +165,19 @@ end
 function VRA:playSpell(spellID)
 	local soundFile = "Interface\\AddOns\\VocalRaidAssistant\\Sounds\\" .. profile.sound.soundpack .. "\\" .. spellID ..
 									".ogg"
+
+	local channel = profile.sound.channel
 	if soundFile then
-		local success = PlaySoundFile(soundFile, addon.SOUND_CHANNEL[profile.sound.channel])
-		if not success and GetCVar("Sound_EnableAllSound") ~= "0" then
-			print(format("VRA - Missing soundfile for configured spell: %s, Voice Pack: %s", GetSpellInfo(spellID),profile.sound.soundpack))
+		local success = PlaySoundFile(soundFile, channel)
+		if not success then
```

Comment:
Won't this spam the user until they turn in sound again?

There could be a reason to temporarily turn off sound. If you can add a buffer-time between each message (say 10 minutes) so we don't spam the user.

The missing for configured spell can be set without any buffer-time if you feel that's more appropriate. The issue with this one if it has a buffer-time is that we would need to track "broken" abilities and buffer-time on them.

Reply:
Valid point, this will produce a lot messages.
We can work with some 'throttling' here.

Reply:
I set up a changelog note for 2.3.2 for when this PR is complete.

Reply:
Added type specific throttling.
For type 'msg' 15s is set, 'sound' is user configured